edenia/tikuna
# Changed concensus node for Prysm node and configurate eth node execut…

…or in Kubernetes

### Title
Prysm node, and configure eth node executor

### What does this PR do?

- Change the lighthouse node for Prysm consensus node, and configure the eth node executor. #45 

### Steps to test
1.Execute the next commands in the Cluster Kubernetes to check the errors at nodes:
kubectl logs eth-executor-0 -n ethereum
kubectl logs prysm-beacon-0 -n ethereum

#### CheckList
